src/test/java/io/pravega/connectors/hadoop/PravegaConnectorLocalJobITCase.java

```diff
@@ -109,7 +109,7 @@ public void testPravegaConnectorInput() throws Exception {
         }
 
         String endPos1 = PravegaInputFormat.fetchLatestPosition(
-                SETUP_UTILS.getControllerUri(), TEST_SCOPE, TEST_STREAM);
+                SETUP_UTILS.getClientConfig(), TEST_SCOPE, TEST_STREAM);
```

Comment:
Looking at here again, I'm not sure if it is giving me an existing ClientConfig (as the name suggests) or initializing a new one, without having to inspect the code in the getClientConfig. I think it'd be simpler if you rename it to something like `initializeClientConfig()` or `prepareClientConfig()` or equivalent.

Reply:
I don't see a problem with the method name and will leave it as-is.